### DISCUSSION TITLE
Linux Support Idea

### DISCUSSION THREAD

**Discussion by Socerest2** [ThioJoe/YT-Spammer-Purge#65] [Ideas]
In response to #54 #51. I was going to make my own build and make a pull request, but since the executables are under releases, I can't. But really what I had in mind was using something like http://www.pyinstaller.org/ to make a linux executable. I know you can just use python on the terminal but having executables shouldn't be exclusive to windows. Just a spitball, what do you all think?

**Comment by ThioJoe** [comment]
Unless I'm wrong, most Linux distributions come with python, and someone who's running Linux should have enough knowledge to be able to run it.  Having to compile yet another binary for Linux would be too much of a pain for me and frankly not necessary I think.

**Comment by Socerest2** [comment]
Yeah - it's really up to you. I agree that most people running linux have the know how, but it may be a good idea to add an mac os executable. I think if there seem to be need/want for it you should add it.

**Comment by ThioJoe** [comment]
It would probably be even harder to distribute an executable on Mac because I don't have an EV code signing certificate. It's locked down hard it would be just as hard to explain how to run the executable as it is to explain how to run it with python. I'll convert this to a discussion but I have no plans to create any additional binaries besides for windows at this time.

**Comment by reesericci** [comment]
I'll look into packaging this as an AppImage, shouldn't be too hard.

**Comment by KendallDoesCoding** [comment]
Are you trying to run the .exe file or are you trying to download the code and then run it? I think Linux has a issue if your trying to download the code and run it, many other people are facing this problem too. The .exe should work just fine though.

@ThioJoe coded this on Windows, and Linux has a certain variation. It would help if someone fixed all the issues and gave it back to @ThioJoe and then he could make another branch called LINUX, for the Linux users who are having problems on this.

**Reply by reesericci** [reply]
.exe doesn't work on linux

**Reply by KendallDoesCoding** [reply]
Oh, I got a solution for that, just need @ThioJoe to merge #118
